libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -415,4 +425,61 @@ void AP_Camera_Backend::log_picture()
 }
 #endif
 
+bool AP_Camera_Backend::set_tracking_external(TrackingType tracking_type, const Vector2f& p1, const Vector2f& p2) {
+#if AP_CAMERA_TRACKING_ENABLED
```

Comment:
```suggestion
#if AP_CAMERA_TRACKING_ENABLED
bool AP_Camera_Backend::set_tracking_external(TrackingType tracking_type, const Vector2f& p1, const Vector2f& p2) {
```

Don't call the method in the first place